src/main/webapp/js/instructorCourseEdit.es6

```diff
@@ -469,8 +469,26 @@ function editFormRequest(e) {
     });
 }
 
+function enableOnclicks() {
+    $(document).on('click', '.addSectionLevelForInstructor', function () {
+        const index = $(this).attr('data-index');
+        const firstBlankSectionRowIndex = $(this).attr('data-first-blank-section-row-index');
```

Comment:
The problem with the `attr` vs `data` is that `data` decides to act smart and cast the type from a string to whatever it thinks it should be. I think I can work around this, though.